ArmA.Studio/App.xaml.cs

```diff
@@ -126,6 +126,21 @@ public static bool SwitchWorkspace()
             }
             var workspace = dlgDc.CurrentPath;
             ConfigHost.App.WorkspacePath = workspace;
+
+            //Save new Workspace to Prev Workspace List
+            List<string> prevWorkSpaces = ConfigHost.App.PrevWorkspacePath;
+            if (prevWorkSpaces.Contains(workspace))
+            {
+                prevWorkSpaces.Remove(workspace);
+            }
+            prevWorkSpaces.Insert(0, workspace);
+            int numSpaces = 5;
+            if(prevWorkSpaces.Count > numSpaces)
+            {
+                prevWorkSpaces.RemoveRange(numSpaces, (prevWorkSpaces.Count- numSpaces));
+            }
+            ConfigHost.App.PrevWorkspacePath = prevWorkSpaces;
+
```

Comment:
Should be moved into the OK command in `ArmA.Studio/Dialogs/WorkspaceSelectorDialogDataContext.cs`